ksql-engine/src/test/java/io/confluent/ksql/structured/SchemaKTableTest.java

```diff
@@ -162,4 +186,62 @@ public void testFilter() throws Exception {
                       initialSchemaKTable);
   }
 
+  @Test
+  public void testGroupBy() {
+    String selectQuery = "SELECT col0, col1, col2 FROM test1;";
```

Comment:
Isn't test1 a stream? Should this be test2? Or are you testing groupBy on streams? Or does it not matter what you are selecting from.. all you are interested in is the schema of the source node?

Reply:
Good catch. Copypasta from the other tests. The tests are still valid since the schema happens to be compatible. But we should use the right source here. Fixing it.